pages/mi/mi-github-and-repositories.md

```diff
@@ -51,11 +51,47 @@ The previous step created a clone of your repository on your OS.
 
 ![GitHub Clone URL](image/mi-forking-and-updating-a-repo.png)
 
-Now, there are three different Github repository levels: [open-learning-exchange.github.io](https://github.com/open-learning-exchange/open-learning-exchange.github.io), your username.github.io on GitHub, and your username.github.io on your OS. These three levels need to be constantly synced and up to date with one another as we will all be contributing to the upstream repository (open-learning-exchange.github.io). It's important to try and keep these separate and avoid mixing changes between them, as you will be unable to fork and git push/pull if they are very different versions.
```

Comment:
I think this first part is pretty important to explain. I suggest leaving this in as an explanation as it allows potential interns to learn information like why the repos need to be synced.